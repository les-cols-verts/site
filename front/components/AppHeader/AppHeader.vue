<template>
  <header class="app-header">
    <Logo />
    <h1 class="title">
      Les Cols verts&#160;: pollinisateurs<br />d'agriculture urbaine
    </h1>
    <div class="donation"></div>
  </header>
</template>

<script>
import Logo from './components/Logo';

export default {
  components: { Logo },
};
</script>

<style>
.app-header {
  margin: 8px;
  display: flex;
  align-items: center;
  justify-content: space-evenly;
}

.title {
  font-family: 'Quicksand', 'Source Sans Pro', -apple-system, Arial, sans-serif;
  display: block;
  font-weight: 300;
  font-size: 16px;
  color: #35495e;
  letter-spacing: 1px;
}

.donation {
  display: none;
}

@media only screen and (min-width: 640px) {
  .app-header {

  }

  .title {
    font-size: 30px;
  }
}

@media only screen and (min-width: 1024px) {
  .app-header {

  }

  .title {
    font-size: 40px;
  }

  .donation {
    display: block;
    width: 250px;
  }
}
</style>
