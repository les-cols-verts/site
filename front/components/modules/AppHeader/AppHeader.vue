<template>
  <header class="app-header">
    <Logo />
    <h1 class="title">
      Les Cols verts&#160;: pollinisateurs<br />d'agriculture urbaine
    </h1>
    <div class="donation"></div>
  </header>
</template>

<script>
import Logo from './components/Logo';

export default {
  components: { Logo },
};
</script>

<style>
.app-header {
  padding: 8px;
  display: flex;
  align-items: center;
  justify-content: space-evenly;
  background-color: #fff;
}

.title {
  display: block;
  font-size: 16px;
  color: #35495e;
  letter-spacing: 1px;
}

.donation {
  display: none;
}

@media only screen and (min-width: 640px) {
  .title {
    font-size: 30px;
  }
}

@media only screen and (min-width: 1024px) {
  .title {
    font-size: 40px;
  }

  .donation {
    display: block;
    width: 250px;
  }
}
</style>
