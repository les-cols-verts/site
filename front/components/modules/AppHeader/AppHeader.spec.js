import AppHeader from '@/components/modules/AppHeader/AppHeader';

describe('AppHeader', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallowMount(AppHeader);

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
