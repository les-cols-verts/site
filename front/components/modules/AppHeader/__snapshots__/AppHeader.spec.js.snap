// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should match snapshot 1`] = `
<header class="app-header">
  <logo-stub></logo-stub>
  <h1 class="title">
    Les Cols verts&nbsp;: pollinisateurs<br>d'agriculture urbaine
  </h1>
  <div class="donation"></div>
</header>
`;
